controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -112,6 +113,55 @@ func buildArgs(desired *flowsv1alpha1.FlowCollectorConsolePlugin, desiredLoki *f
 }
 
 func (b *builder) podTemplate(cmDigest string) *corev1.PodTemplateSpec {
+	volumes := []corev1.Volume{{
+		Name: secretName,
+		VolumeSource: corev1.VolumeSource{
+			Secret: &corev1.SecretVolumeSource{
+				SecretName: secretName,
+			},
+		},
+	}, {
+		Name: configVolume,
+		VolumeSource: corev1.VolumeSource{
+			ConfigMap: &corev1.ConfigMapVolumeSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: configMapName,
+				},
+			},
+		},
+	},
+	}
+
+	volumeMounts := []corev1.VolumeMount{{
+		Name:      secretName,
+		MountPath: "/var/serving-cert",
+		ReadOnly:  true,
+	},
+		{
+			Name:      configVolume,
+			MountPath: configPath,
+			ReadOnly:  true,
+		}}
+
+	args := []string{
+		"-cert", "/var/serving-cert/tls.crt",
+		"-key", "/var/serving-cert/tls.key",
+		"-loki", querierURL(b.desiredLoki),
+		"-loki-labels", strings.Join(constants.LokiIndexFields, ","),
+		"-loki-tenant-id", b.desiredLoki.TenantID,
+		"-loki-skip-tls", "true",
```

Comment:
```suggestion
```

Reply:
Do you also experience [this issue](https://issues.redhat.com/browse/NETOBSERV-485) on args order ?

Reply:
Either I was lucky and it kept the same order, or I did not have the issue.

Using this configuration I was able to reach a loki with tls:
`    tls:
      enable: true
      insecureSkipVerify: true
`